gnd/src/main/res/layout/photo_field.xml

```diff
@@ -39,5 +39,14 @@
       android:contentDescription="@string/photo_preview"
       android:scaleType="fitCenter"
       app:imageUri="@{viewModel.uri}" />
+    <ImageButton
+      android:layout_width="wrap_content"
+      android:layout_height="wrap_content"
+      android:layout_gravity="top|end"
+      android:onClick="@{() -> viewModel.clearResponse()}"
+      android:visibility="@{viewModel.getClearButtonVisibility()}"
+      app:backgroundTint="@android:color/transparent"
+      app:srcCompat="@drawable/ic_baseline_cancel_24"
```

Comment:
I believe this icon is usually used to close a popup or other UI; can we use the Material [trash icon](https://material.io/resources/icons/?icon=delete&style=baseline) here instead? Open to other suggestions.